### PR TITLE
Group 3

### PR DESCRIPTION
**Pull request by hewittk**
Suggestions for all criteria in the Continuous Integration with Github Actions section.

### CODE REVIEW

File: assessment.md [hunk 1]
```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
...
+  * Excellent = Streamlines all major angles including testing and general communication using Github Actions.
 
 * Performing secure releases of project deliverables
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not set up any security features on project deliverables.
+  * Inadequate = Sets up some security features on project deliverables, but project information is still easily compromisable.
+  * Adequate = Properly sets up all appropriate security features on project deliverables.
+  * Good = Properly sets up all appropriate security features on project deliverables in way that is well-suited for given project deliverables.
+  * Excellent = Properly sets up all appropriate security features on project deliverables in way that is best-suited for given project deliverables, and actively monitors project for potential security threats or improvements.
```
Comment (enpuyou) [inline assessment.md]: The use of "security features" in this section is ambiguous. It would be more helpful if provided with examples of what can be counted as a secure (or insecure) feature. I also encourage the team to do a little research on how to create a so-called "secure" release.

File: assessment.md [hunk 2]
```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
...
+  * Good = Often times integrates third party software with Github. Very knowledgeable on how to do so.
+  * Excellent = Leads the way for the team when integrating Github with a third party. Can easily explain to the team how they did so.
 
 * Adding README badges that show status of GitHub Actions CI builds
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not add README badges.
+  * Inadequate = Rarely adds README badges that show status of builds, never addressing any errors made.
+  * Adequate = Occasionally adds README badges that show status of builds, rarely addressing errors any errors made.
+  * Good = Frequently adds README badges that show status of builds, often addressing any errors made.
```
Comment (enpuyou) [inline assessment.md]: This assessment is to some degree unrealistic. Often time, there are only very few badges that could be added to reflect the build status. "Frequently adds badges" would seem impossible. It would be best for the team to come up with other assessments on the skill of using README badges.

File: assessment.md [hunk 3]
```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
...
+  * Excellent = Adds README badges that show project characteristics, always addressing any errors made.
 
 * Identifying and resolving problems with failing builds in GitHub Actions CI
-
-TODO: add description for each assessment level for this skill.
+  * None = Ignores all problems in GitHub Actions CI, routinely makes and does not resolve basic coding errors.
+  * Inadequate = Occasionally identifies problems with builds, does not solve problems.
+  * Adequate = Frequently identifies problems, occasionally solves them partially.
+  * Good = Frequently identifies problems and makes strides to solve them at least partially. Occasionally provides documentation.
+  * Excellent = Routinely identifies and solves the problems behind failing builds and documents any fixes for future use.
```
Comment (enpuyou) [inline assessment.md]: Instead of assessing the frequency of "identifying and resolving problems", I would advise the team to maybe address the percentage of problems that an individual is capable of identifying and resolving, i.e. identify some problems, identify most problems. Also, I think it would be great if you could be consistent with the use of "resolve" and "solve".

File: assessment.md [hunk 4]
```diff
@@ -119,29 +119,47 @@ TODO: add description for each assessment level for this skill.
 ### Continuous Integration with Github Actions
 
 * Setup and configuration of GitHub Actions CI
-
-TODO: add description for each assessment level for this skill.
+  * None = Does not use Github Actions CI or have it set up on their repository.
+  * Inadequate = Set up Github Actions to a workflow that doesn't make sense with the program and its contributors or in a way that isn't functionally committed to the repository.
+  * Adequate = Properly set up a Github Actions workflow in a way that is usable to collaborators.
+  * Good = Set up a Github Actions workflow that streamlines the workflow process and makes process consistent among devices that have different installations.
```
Comment (enpuyou) [inline assessment.md]: "Set up a Github Actions workflow that streamlines the workflow process" seems a bit wordy.

**Review by enpuyou — CHANGES_REQUESTED** [review]
Hi, members of group 3, I have reviewed your proposed change and the detailed feedback can be found below. I have just updated your branch with the assessment format the class has agreed on (using emojis). In general, try to stay consistent with the use of third-person singular (in the `Setup and configuration of Github Actions CI` section), as well as the use of period at the end of each assessment. Thanks!

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Aside from @enpuyou 's great suggestions, I'd like to mention that an emoji scaling system was recently merged into main. This means that either all groups should adopt it and add the needed changes, OR propose new emojis/symbols that need to be agreed upon. Either choice is fine but please make sure that your PR is reflecting your choice by changing the following keywords on each assessment bullet point.

```
None
Inadequate
Adequate
Good
Excellent
```

**Review by noorbuchi — APPROVED** [review]
Thanks for addressing my requested changes. I will approve this PR but please make sure that all of @enpuyou 's requests are added as well

**Review by corlettim — APPROVED** [review]

**Review by enpuyou — APPROVED** [review]
